### DISCUSSION TITLE
Database Shema

### DISCUSSION THREAD

**Discussion by usman-arham** [netbox-community/netbox#8164] [General]
is there a database schema or ER diagram for Netbox,

I need to compare it with other existing tools.

**Comment by DanSheps** [comment]
No, but you may be able to install a development instance and generate a diagram from postgres using DB development tools.

You can view the model definitions within either the models.py or models folder under the following apps: `circuits, dcim, extras, ipam, tenancy, users, virtualization, wireless`

**Comment by ttarabula** [comment]
this should work:
https://django-extensions.readthedocs.io/en/latest/graph_models.html

(install/activate django-extensions and graphviz/pygraphviz, then run `manage.py graph_models -a -g -o netbox-erd.png`)
